main.cpp

```diff
@@ -301,47 +422,71 @@ struct Body
     void makeGuitarLookCool (SongWriter rockWriterA, bool colourfulFinish = true);
 };
 
-/*
-Thing 7) Bridge
-5 properties:
-    1) number of string slot (int)
-    2) material type (std::string)
-    3) color (std::string)
-    4) whammy bar type (std::string)
-    5) number of body screw hole (int)
-3 things it can do:
-    1) hold strings still
-    2) make pitch dive
-    3) match the body color
-*/
+void Body::holdPartsTogether(std::string boltType)
+{
+    if (boltType == "bolt on")
+    {
+        thicknessInMM += 10;
+    }
+}
+
+void Body::improveTheVibration(float resonateFrequency)
+{
+    if(woodType == "alder")
+    {
+        resonateFrequency = resonateFrequency * 1.5f;
+    }
+}
+
+void Body::makeGuitarLookCool(SongWriter rockWriterA, bool colourfulFinish)
+{
+    if(colourfulFinish)
+    {
+        rockWriterA.increaseFollower();
+    }
+}
+
+
 struct Bridge
 {
     int numberOfStringSlot = 6;
     std::string materialType = "stainless steel";
     std::string color = "nickel";
     std::string whammyBarType = "floyd rose";
     int numberOfBodyScrewHole = 2;
+    bool intonationAdjustable = true;
 
     void holdStringsStill(bool stringInTune = true);
     void makePitchDive(float stringTensionInKg = 1);
-
-    //returns if the bridge matches guitar body's color
-    bool matchTheBodyColor(SongWriter rockWriterA);
+    void matchTheBodyColor(SongWriter rockWriterA);
 };
 
-/*
-Thing 8) String
-5 properties:
-    1) wound type (std::string)
-    2) 1st string gauge in inch (float)
-    3) 6th string gauge in inch (float)
-    4) brand (std::string)
-    5) coat type (std::string)
-3 things it can do:
-    1) vibrate
-    2) get rust
-    3) get broken
-*/
+void Bridge::holdStringsStill(bool stringInTune)
+{
+    if(intonationAdjustable)
+    {
+        stringInTune = true;
+    }
+    else
+    {
+        stringInTune = false;
+    }
+}
```

Comment:
What is the point of passing in a function argument, if you're just going to overwrite the value? 
- [ ] revise

Reply:
Yes, I want to overwrite the value then use this value later in other function, if so, should I make this function return bool then return this value?